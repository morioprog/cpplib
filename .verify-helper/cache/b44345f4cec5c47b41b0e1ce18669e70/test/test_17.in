6
-1 1
-8 9
-2 3
-1 4
-2 5
0 5
