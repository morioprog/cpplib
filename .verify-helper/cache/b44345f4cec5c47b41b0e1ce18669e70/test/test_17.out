0 5
-1 4
-2 5
-2 3
-8 9
-1 1
